src/routing.test.js

```diff
@@ -14,6 +15,10 @@ function locator(data) {
 }
 
 describe('parsePath()', () => {
+  afterAll(() => {
+    web3Providers.default.disconnect()
```

Comment:
Yeah... this is a part that really bugs me but I guess we can fix it here first and then solve it for real later.

I have a 3 month old branch I've been meaning to clean up and push so that our import dependency tree doesn't do crazy things like start a connection to an ETH node on start up.

Reply:
Yes, that’s what I was thinking for the next step. Looking forward to this PR :eyes: